libs/platform/src/lib/form/combobox/combobox/combobox.component.ts

```diff
@@ -227,6 +227,17 @@ export class ComboboxComponent extends BaseCombobox implements ComboboxInterface
         this.isOpenChangeHandle(false);
     }
 
+    /** @hidden */
+    _getLabelledBy(): Nullable<string> {
+        let retVal = this.ariaLabelledBy;
+
+        if (this.stateMessage) {
+            retVal = retVal + ' fdp-list-message-combobox-state-message-' + this.id;
+        }
+
+        return retVal;
+    }
```

Comment:
this can be simplified:
```
_getLabelledBy(): Nullable<string> {
    return this.stateMessage
        ? `${this.ariaLabelledBy} fdp-list-message-combobox-state-message-${this.id}`
        : this.ariaLabelledBy;
}
```